netbox/wireless/tables/wirelesslink.py

```diff
@@ -36,6 +37,10 @@ class WirelessLinkTable(TenancyColumnsMixin, NetBoxTable):
         verbose_name=_('Interface B'),
         linkify=True
     )
+    length = columns.TemplateColumn(
+        template_code=WIRELESS_LINK_LENGTH,
+        order_by=('_abs_length', 'length_unit')
```

Comment:
Absolute length is stored in meters; it doesn't make sense to also order by the original unit. (I don't know why this is set on the column in CableTable; it should also be removed.)
```suggestion
        order_by=('_abs_length')
```